### DISCUSSION TITLE
Feedback floccus.org website

### DISCUSSION THREAD

**Discussion by marcelklehr** [floccusaddon/floccus#1031] [General]
Hey there!

this post is all about https://floccus.org .

How would you rate it on a scale from 1-5? Which aspects bother you? Is there something you think is missing, or does something not belong on there in your opinion? Were there parts that could be more useful when starting out with or getting to know more about floccus? If you've been a user for a while already, is it still useful to you?

If you have some time, I'd love to hear your feedback :)

**Comment by pinpontitit** [comment]
Yes I have one thing in my mind since two weeks:
the text in the headers is not correctly wrapped.

Normally it should be wrapped between two words, and on the floccus website it is wrapped in the middle of a word.

Example:

_This is floc
cus. A grea
t app for sy
ncing book
marks._

The worst is in the FAQ page, but it happens in others too once in a while.

EDIT: this happens when reading from my phone browser. Maybe it wraps properly from desktop browser.


Besides that, I didnt explored, nor really read, so much yet. 

The only thing I can say is that the design is beautifully made !
Is it also made with Vuetify components or other framework is used ?

![Screenshot_20220531-022727_Chrome.jpg](https://user-images.githubusercontent.com/100489443/171071653-9897b26a-cedc-4cf5-8eec-dc08a0538e70.jpg)

**Reply by marcelklehr** [reply]
huh, I will investigate
